src/h5web/toolbar/controls/ColorMapSelector/ColorMapSelector.module.css

```diff
@@ -2,6 +2,12 @@
   display: flex;
 }
 
+.option {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
```

Comment:
This style was previously named `.colormap` and located in `Selector.module.css`. As a result it was being applied to the scale selector's options as well, which explains why the labels were spaced out from their icons.

Reply:
:fishing_pole_and_fish: